### PR TITLE
Fanatic Support LA Mirror 

### PR DESCRIPTION
**Pull request by fanaticsupport**
Fanatic Support Los Angeles/Pasadena Mirror

### CODE REVIEW

File: mirrors.d/almalinux.fanaticsupport.io.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+sponsor_url: https://fanaticsupport.io
+email: leo@fanaticsupport.io
+geolocation:
+  continent: Southern California
+  country: US
+  state_province: California
+  city: Los Angeles
+private: false
+monopoly: false
+asn: 174
```
Comment (jonathanspw) [inline mirrors.d/almalinux.fanaticsupport.io.yml:17]:
```suggestion
```

Comment (jonathanspw) [reply mirrors.d/almalinux.fanaticsupport.io.yml:17]: This would direct all traffic on Cogents network to your mirror.  Definitely not what you want here.

**Review by jonathanspw — CHANGES_REQUESTED** [review]

**Comment by fanaticsupport** [comment]
Change has been done.

**Review by jonathanspw — APPROVED** [review]

**Comment by jonathanspw** [comment]
Thanks for the mirror!

**Pull request merged**